sys/net/gnrc/sock/udp/gnrc_sock_udp.c

```diff
@@ -278,6 +279,14 @@ ssize_t sock_udp_send(sock_udp_t *sock, const void *data, size_t len,
     }
     else {
         rem = (sock_ip_ep_t *)remote;
+#if GNRC_NETIF_NUMOF == 1
+        /* set interface implicitly */
+        gnrc_netif_t *netif = gnrc_netif_iter(NULL);
+
+        if (netif != NULL) {
+            rem->netif = netif->pid;
```

Comment:
If I read the code correctly it is possible that this changes the `const sock_udp_ep_t *remote` argument here (first assigning `remote` to `rem` and then changing rem here). 

Besides changing the argument to non const, I don't really have a solution, but I think it is a bit ugly to modify something that is passed as `const`

Reply:
Mhhh change it to non-const is of the table, but you are right! This is bad. I try to come up with a better solution.

Reply:
Done